### DISCUSSION TITLE
Scrum Prep: Milestone 3 Week 4

### DISCUSSION THREAD

**Discussion by YanehRuiz** [uprm-inso4116-2024-2025-s1/semester-project-SafeRUM#414] [Weekly Retrospective]
Weekly Development Update: Fourth week of Milestone 3

Hello team!

Please provide a brief comment detailing what you accomplished this week, including any challenges faced or insights gained. Make sure to submit your updates by the deadline, which is before: Friday, November 22th, 2024 @ 11 am

Thank you for your collaboration!

As for my collaboration:

I've been unable to be as present as usual fue to lack of internet.

**Comment by ghost** [comment]
Other than the first sprint task, I haven't been able to work on the project due to other tests, but this weekend I'll make sure to finish my tasks for the milestone.

**Comment by JorgeC08** [comment]
Already completed my last 2 LTT, looking for a place to put it on the documentation. Already working on my other documentation task.

**Comment by noel-vargas** [comment]
I did not have much time to work on the issues this week

**Comment by johnsantiago9** [comment]
Created tasks for milestone and researched about documentation tasks.

**Comment by JorgeRivera94** [comment]
Facing a compatibility issue that may be on my end. Will continue to progress my tasks once resolved.

**Comment by javier-rivera8** [comment]
This week, I was loaded with other classes and exams, so I didn’t make much progress on my tasks.

**Comment by lex939** [comment]
Getting ready to present documentation for M3 and working on formatting especially (legend).

**Comment by Alejandro-DeJesus** [comment]
Didn't do much this week, in hopes of making progress during the weekend.

**Comment by jonVargas1** [comment]
This week, I worked on a lecture topic task and brainstormed ideas for additional lecture topic tasks.

**Comment by williammartinez10** [comment]
Didn’t have time to work on the project this week due to personal matters, but I will be completing my remaining tasks by the end of M3.

**Comment by NathBat09** [comment]
Working on lecture topic tasks

**Comment by RaquelQuinones** [comment]
Didn't do much this week due to hell week.

**Comment by misaelmercado1** [comment]
Worked on Lecture topic task and documentation

**Comment by CaribbeanCool** [comment]
Worked on issues #385 #409 and #410

**Comment by joel-alvarado** [comment]
Did not work on the project due to other classwork.

**Comment by JuanJRivera** [comment]
I did not have much time to work on the issues this week.

**Comment by Jjulianirizarry24** [comment]
Fixed a merge conflict

**Comment by JoseLQV** [comment]
Created more LTT

**Comment by Chosen-Juan1** [comment]
Started working on the remaining lecture topic tasks.

**Comment by Rodal1** [comment]
Report button in Admin Calendar Page
